tests/resources/inputs/calibration/input.json

```diff
@@ -190,60 +190,62 @@
     },
     "overrides": null
   },
-  "pixelGroupingParameters": [
-    {
-      "groupID": 0,
-      "twoTheta": 2.1108948177427838,
-      "dResolution": {
-        "minimum": 0.40224298673776404,
-        "maximum": 2.0112149336888203
-      },
-      "dRelativeResolution": 0.002771822370612031
-    },
-    {
-      "groupID": 1,
-      "twoTheta": 1.82310673131693,
-      "dResolution": {
-        "minimum": 0.44278229953817677,
-        "maximum": 2.213911497690884
+  "pixelGroup": {
+    "pixelGroupingParameters": [
+      {
+        "groupID": 0,
```

Comment:
Group IDs of `0` are considered "unset" by mantid and are masked.  This is a dumb thing mantid does that I complain about constantly, but fixing it is outside the scope of any work we have planned.  In the meantime, increment all of these group IDs so they run 1-6